files/files.go

```diff
@@ -140,35 +187,106 @@ func (c *Content) Sys() interface{} {
 	return nil
 }
 
-// ExpandContentGlobs gathers all of the real files to be copied into the package.
+func (c *Content) String() string {
+	var properties []string
+	if c.Source != "" {
+		properties = append(properties, "src="+c.Source)
+	}
+	if c.Destination != "" {
+		properties = append(properties, "dst="+c.Destination)
+	}
+	if c.Type != "" {
+		properties = append(properties, "type="+c.Type)
+	}
+	if c.Packager != "" {
+		properties = append(properties, "packager="+c.Packager)
+	}
+	if c.FileInfo != nil {
+		if c.FileInfo.Owner != "" {
+			properties = append(properties, "owner="+c.FileInfo.Owner)
+		}
+		if c.FileInfo.Group != "" {
+			properties = append(properties, "group="+c.FileInfo.Group)
+		}
+		if c.Mode() != 0 {
+			properties = append(properties, "mode="+c.Mode().String())
+		}
+		if !c.ModTime().IsZero() {
+			properties = append(properties, "modtime="+c.ModTime().String())
+		}
+		properties = append(properties, "size="+strconv.Itoa(int(c.FileInfo.Size)))
+	}
+
+	return fmt.Sprintf("Content(%s)", strings.Join(properties, ","))
+}
+
+// ExpandContentGlobs gathers all of the real files to be copied into the
+// package.
+//
+// Deprecated: ExpandContentGlobs is deprecated in favor of PrepareForPackager
+// which also expands implicit directories and takes the packager into account.
 func ExpandContentGlobs(contents Contents, disableGlobbing bool) (files Contents, err error) {
```

Comment:
does nfpm still uses this somewhere? if not, I think we can remove it... if anyone is using this, they likely shouldn't. 

on the same note, for a future PR/release, we might move this into the `internal` package.

Reply:
How serious are we about a stable library API? If you want I can remove it, then we can also get rid of all the tests for `ExpandContentGlobs`.

Reply:
I think we can remove it yes...